src/Wombo.Api/Controllers/UsersController.cs

```diff
@@ -19,6 +19,7 @@ public UsersController()
                 new User(2, "Bruna", "bruna@email.com" ),
                 new User(3, "Iran", "iran@email.com"),
                 new User(4, "Rogério", "rogerio@email.com"),
+                new User(5, "Ricardo", "ricardo@email.com"),
```

Comment:
```suggestion
                new User(5, "Ricardo", "ricardo@email.com"),
```
coloca mias 2 usuários